README.md

```diff
@@ -139,6 +139,18 @@ A style guide for Sightly, the HTML templating system from Adobe Experience Mana
         ...
     </sly>
     ```
+    
+  - [3.4](#3.4) <a name='3.4'></a> **Always provide i18n translation hints.**
+  
+  The more context the translators have about a specific text, the better they can do their work.
```

Comment:
I thought the hints were just placed as extra comment information. And if we would change the rule to:

**Always provide i18n translation hints for non-descriptive text.**

That way you are sure it's not text that could leverage out of the box translation. Like that, do you think it make sense to add this new rule, or would you leave it out?

Thanks for the feedback!